sdk/core/core-https/src/xhrHttpsClient.ts

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/// <reference lib="dom" />
+
+import { AbortError } from "@azure/abort-controller";
+import {
+  HttpsClient,
+  PipelineRequest,
+  PipelineResponse,
+  TransferProgressEvent,
+  HttpHeaders
+} from "./interfaces";
+import { RestError } from "./restError";
+import { createHttpHeaders } from "./httpHeaders";
+
+function isReadableStream(body: any): body is NodeJS.ReadableStream {
+  return body && typeof body.pipe === "function";
```

Comment:
just return `false`?

Reply:
hmm, is it possible to have Node streams when running in browsers?

Reply:
Not true node streams, but it's possible that someone could bundle with a polyfill that would implement the stream interface.

Since this method is only used to throw at runtime, I think it's safe?

Reply:
Yes it is safe.